sqlair.go

```diff
@@ -124,14 +129,18 @@ func (db *DB) Query(ctx context.Context, s *Statement, inputArgs ...any) *Query
 	return &Query{pq: pq, run: run, ctx: ctx, err: nil}
 }
 
-// Run is an alias for Get that takes no arguments.
+// Run is an alias for Get that takes no arguments. Run is used to run a query
+// on a database and disregard any results.
 func (q *Query) Run() error {
 	return q.Get()
 }
 
-// Get runs the query and decodes the first result into the provided output arguments.
-// It returns ErrNoRows if output arguments were provided but no results were found.
-// An &Outcome{} variable may be provided as the first output variable.
+// Get runs the query and decodes the first result into the provided output
```

Comment:
```suggestion
// Get runs the query and decodes the first row returned into the provided output
```

Reply:
I think result is normally used to get the statistics like when running `sql.Exec`.